app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -111,29 +127,40 @@ class BrowserActivity : DuckDuckGoActivity(), BookmarkDialogCreationListener, We
         configureObservers()
         configureToolbar()
         configureWebView()
+        viewModel.registerWebViewListener(webViewClient, webChromeClient)
         configureOmnibarTextInput()
         configureFindInPage()
-        configureDummyViewTouchHandler()
         configureAutoComplete()
+    }
 
-        if (savedInstanceState == null) {
```

Comment:
Does removing this break sharing from an external app (e.g share a ink from Chrome) to our app if our app isn't already running?